Small fonts appeared too small

loicgilbert opened the pull request.

loicgilbert committed:
Small fonts appeared too small

pavel-kirienko commented on kucher/view/utils.py:
```diff
@@ -78,7 +78,7 @@ def get_monospace_font(small=False) -> QFont:
 def _get_monospace_font_impl(small=False) -> QFont:
     begun = time.monotonic()
-    multiplier = 0.8 if small else 1.0
+    multiplier = 0.9 if small else 1.0
```
What I was referring to in the issue description is that while the font was too small on Windows, it was of just the right size on GNU/Linux systems. So I am thinking that we should constrain the absolute size of the selected font instead of changing the multiplier. Perhaps like `max(12, QFont().pointSize() * multiplier)`, where 12 should be defined as a constant somewhere in this module, and its value should be determined empirically.

loicgilbert committed:
Defined minimum font size

That should not affect font size on Linux, while still make it bigger on windows

pavel-kirienko commented:
Cool. Do you happen to have screenshots on Windows and Ubuntu so that we could compare how they look? FYI in my log files, I found this:

```
2019-05-03 10:27:34,274 pid=8063  INFO     kucher.view.utils: Selected monospace font (0.000017 seconds): 'Consolas,8,-1,5,50,0,0,0,0,0'
```

We can see here that on KDE the selected font size is 8, not sure if it matters though.

loicgilbert commented:
On windows:
![windows](https://user-images.githubusercontent.com/51403458/59185166-6fd3d200-8b78-11e9-9f18-b2bc498dd1fa.png)

On linux:
![linux](https://user-images.githubusercontent.com/51403458/59185196-80844800-8b78-11e9-841d-41f762d87f37.png)

pavel-kirienko commented:
Okay, good. There is one corner case I should have thought of earlier: if `QFont().pointSize()` is less than 7pt (which is unlikely but possible), the returned font will be always larger than the default. I suppose it could be trivially fixed as `min_font_size = min(7, QFont().pointSize())`. After that last thing is accounted for this PR should be good to merge.

loicgilbert commented:
Yes but in that case, all fonts will be same size, `multiplier` will be useless.

loicgilbert committed:
Adapted minimum font size
